### ISSUE TITLE
LCD display

### ISSUE THREAD

**Issue by ksodib** [MythTV-Clients/MythTV-Android-Frontend#86]
Is it possible to implement LCD display function in Android Frontend? Just asking.

**Comment by billmeek** [comment]
Are you referring to the output of  MythTV's `mythlcdserver`?

**Comment by dmfrey** [comment]
We are limited by what is available in the myth services api. If the
corresponding commands are available, then it is something we can add to
the app as an enhancement.

I am not familiar with this functionality. Could you provide an example of
what you would like to have added?

**Comment by ksodib** [comment]
I think about playing music without TV monitor as is explained here:
http://www.mythtv.org/wiki/LCDproc
My backend is without LCD display and i think it could be nice to have it
on  mythmote part of MythTV Android Frontend

2012/11/25 Daniel Frey notifications@github.com

> We are limited by what is available in the myth services api. If the
> corresponding commands are available, then it is something we can add to
> the app as an enhancement.
> 
> I am not familiar with this functionality. Could you provide an example of
> what you would like to have added?

**Comment by ksodib** [comment]
Yes. Is it possible to have it on mythmote of MythTV Android Frontend?

2012/11/25 billmeek notifications@github.com

**Comment by pot8oe** [comment]
It probably will not use the LCD specific functionality but a similar
interface might be do-able through the service API.

-Tom
